### PR TITLE
Remove this system role to SLERT

### PR DESCRIPTION
**Pull request by JeffreyCheung**

### CODE REVIEW

File: package/system-role-text-mode.changes [hunk 1]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Jan 11 04:23:56 UTC 2019 - Jeffrey Cheung <jcheung@suse.com>
+
+- Remove this system role to SLERT, I am working on a new
```
Comment (jreidinger) [inline package/system-role-text-mode.changes:4]: remove this role *from* SLERT

**Comment by JeffreyCheung** [comment]
Please help to review

**Comment by imobachgs** [comment]
> Please help to review

There is still an small issue in the changes file as pointed out by @jreidinger.

**Comment by imobachgs** [comment]
Ok, to speed up things a bit, I've created #14 which solves the mentioned issues.

**Pull request closed without merging**